src/tequila/circuit/visualize.py

```diff
@@ -0,0 +1,128 @@
+import importlib.util
+
+import tequila
+from typing import Optional, List
+import matplotlib.pyplot as plt
+import matplotlib.figure as figure
+import networkx
+
+
+def visualize(qubit_hamiltonian: tequila.QubitHamiltonian,
+              circuit: Optional[tequila.QCircuit] = None,
+              connectivity: Optional[List[tuple]] = None,
+              file_path: Optional[str] = None) -> figure.Figure:
+    """
+    Precondition:
+    The maximum number of qubits is 10 at the moment (Feb24, 2023)
+
+    Post condition:
+    The graph of the qubit_hamiltonian is displayed
+    or is stored in file_path
+
+    One thing to note is that if you are using command-line interface,
+    the plot might not be successfully shown
+
+    , so it's better idea to save it as an image.
+    === parameters ===
+    qubit_hamiltonian: A QubitHamiltonian representation of pauli operators
+    circuit: A QCircuit that corresponds to the
+    connectivity: networkx.Graph.edges that show the connectivity of qubits
+    A nested list should be provided for this argument.
+    file_name: str format of file name
+    to which the plotted graph will be exported. The file will be a png format
+
+    === return ===
+    None
+
+    === sample usages ===
+    >>> visualize(
+    >>> tequila.QubitHamiltonian(openfermion.QubitOperator("X0 X5 Y3", 1)))
+    *** A graph with nodes 0 and 5 having colour red
+    and node 3 having colour green ***
+    >>> visualize(
+    >>> tequila.QubitHamiltonian(openfermion.QubitOperator("X0 X5 Y3", 1)),
```

Comment:
replace openfermion call with QubitHamiltonian("X(0)X(5)Y(3)"), that's shorter and more robust regarding future changes
or
paulis.X([0,5])*paulis.Y(3)

Reply:
Yes, I've removed all the openfermion related part in visualization that I can abbbreviate